.github/workflows/check-contracts.yml

```
@@ -49,0 +1,59 @@
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  check-contracts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3  # Updated to v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)